resources/privacy-and-gdpr.md

```diff
@@ -192,15 +236,11 @@ To retrieve all the identifier data stored locally by the SDK as a JSON string,
 
```

Comment:
Just a reminder on this: we should update this and provide a link to the Edge identity extension documentation